src/components/PetAttributes.jsx

```diff
@@ -9,21 +9,43 @@ const PetAttributes = props => {
   const { attributes } = props;
 
   var displayAttributes = [
-    "Species",
-    "Breed",
-    "Color",
+    "Type",
+    "Primary Breed",
+    "Primary Color",
     "Sex",
     "Weight",
     "Age",
-    "Spayed/Neutered",
+    "Altered",
     "Shelter Arrival Date"
   ];
 
+  //TODO: Possibly needs to be added to the service. But for now this is where this lives
```

Comment:
Yea I agree, trying to find a good balance of making the servcei flexible but usabe. We can come back to this.

Reply:
adding this to my notes.